lib/DDG/Spice/Syntax.pm

```diff
@@ -0,0 +1,27 @@
+package DDG::Spice::Syntax;
+
+# ABSTRACT: Look up syntax for a programming language
+
+use DDG::Spice;
+
+# Caching - http://docs.duckduckhack.com/backend-reference/api-reference.html#caching
+spice is_cached => 1;
+spice proxy_cache_valid => "200 1d"; # defaults to this automatically
+
+spice wrap_jsonp_callback => 1; # only enable for non-JSONP APIs (i.e. no &callback= parameter)
+
+# API endpoint - http://docs.duckduckhack.com/walkthroughs/forum-lookup.html#api-endpoint
+spice to => 'https://syntaxdb.com/api/v1/concepts/search?q=$1';
+
+# Triggers - https://duck.co/duckduckhack/spice_triggers
+triggers any => 'syntax', 'syntaxdb', 'syntaxcenter';
+
+# Handle statement
+handle remainder => sub {
+
+    # Query is in $_ or @_, depending on the handle you chose...if you
+    # need to do something with it before returning
+    return $_;
```

Comment:
Definitely didn't think of non-programming related syntax queries, I'll work on making a list of keywords and languages and writing a regex.

For `any`, the reason I did that was because if someone searched something like `Java syntax for loop` then I'd want the IA to trigger. I originally did have it as `startend`.